### PR TITLE
Add to google adsense

### PR DESCRIPTION
**Pull request by hsiangfeng**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [X] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes was maked (for bug fixes / features).
   - [ ] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [ ] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [X] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [X] Other... Please describe:

## What is the current behavior?

add to google adsense

Issue resolved: N/A

## What is the new behavior?

- Screenshots with this changes:  https://i.imgur.com/KP0am5r.png
- Link to demo site with this changes: https://hsiangfeng.github.io

### How to use?
In NexT `_config.yml`:
```yml
# Google Analytics
#google_analytics:
#  tracking_id:
#  localhost_ignored: true
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [X] No.

### CODE REVIEW

**Comment by hsiangfeng** [comment]
Google AdSense is applying ... :(

**Comment by 1v9** [comment]
Is this feature necessary?

**Comment by hsiangfeng** [comment]
I don't know, because some of my friends want to use Google AdSense.
Unfortunately, now Google AdSense is applying ,Can't see the actual effect... :(

**Comment by 1v9** [comment]
@hsiangfeng Actually Google just refused my application a moment ago.

**Comment by jiangtj** [comment]
image
广告有很大的可定制性，所以我们很难做个通用的给其它用户使用，如果需要添加，这个Google广告的自动广告就足够了
Ads are very customizable, so it's hard to make a generic AD for other users. If you need to add it, this Google auto AD will be enough

File: layout/_third-party/analytics/google_adsense.swig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if theme.google_adsense.google_ad_client_id %}
+<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+<script>
+     (adsbygoogle = window.adsbygoogle || []).push({
+          google_ad_client: {{ theme.google_adsense.google_ad_client_id }},
+          enable_page_level_ads: {{ theme.google_adsense.enable_page_level_ads }}
+     });
+</script>
+{% endif %}
```
Comment (jiangtj) [inline layout/_third-party/analytics/google_adsense.swig]: 在一个地方添加即可，添加的位置不代表广告出现的位置，建议放在head中
Just add it in one place. The added location does not represent the location where the AD appears. It is recommended to put it in the head

Comment (hsiangfeng) [reply layout/_third-party/analytics/google_adsense.swig]: This is also a way

Comment (jiangtj) [reply layout/_third-party/analytics/google_adsense.swig]: But you put it in both sidebar and post.

Comment (jiangtj) [reply layout/_third-party/analytics/google_adsense.swig]: If you want to ads appear in sidebar, try '广告单元'. 
You copied the wrong code :joy: 
https://support.google.com/adsense/answer/9190028?hl=zh-Hans&ref_topic=28893

**Pull request closed without merging**